test/services/BulletPointBuilderSpec.scala

```diff
@@ -44,63 +44,63 @@ class BulletPointBuilderSpec extends BaseSpec with ProcessJson {
 
       val text = ""
 
-      asString(BulletPointBuilder.fragmentsToDisplayAsSeq(text)) shouldBe text
+      asString(TextBuilder.flattenPlaceholders(text)) shouldBe text
```

Comment:
It would be good to add some tests with link-same and link-tab to confirm parsing of new format links.